src/main/java/org/jabref/gui/SaveOrderConfigDisplay.java

```diff
@@ -38,56 +46,87 @@ private void init() {
         fieldNames.add(BibEntry.KEY_FIELD);
         Collections.sort(fieldNames);
         String[] allPlusKey = fieldNames.toArray(new String[fieldNames.size()]);
-        savePriSort = new JComboBox<>(allPlusKey);
+        savePriSort = new ComboBox<>(FXCollections.observableArrayList(allPlusKey));
         savePriSort.setEditable(true);
-        saveSecSort = new JComboBox<>(allPlusKey);
+        saveSecSort = new ComboBox<>(FXCollections.observableArrayList(allPlusKey));
         saveSecSort.setEditable(true);
-        saveTerSort = new JComboBox<>(allPlusKey);
+        saveTerSort = new ComboBox<>(FXCollections.observableArrayList(allPlusKey));
         saveTerSort.setEditable(true);
 
-        savePriDesc = new JCheckBox(Localization.lang("Descending"));
-        saveSecDesc = new JCheckBox(Localization.lang("Descending"));
-        saveTerDesc = new JCheckBox(Localization.lang("Descending"));
-
+        savePriDesc = new CheckBox(Localization.lang("Descending"));
+        saveSecDesc = new CheckBox(Localization.lang("Descending"));
+        saveTerDesc = new CheckBox(Localization.lang("Descending"));
+
+        Font font = new Font(10);
+        GridPane builder = new GridPane();
+        Label label = new Label(Localization.lang("Primary sort criterion"));
```

Comment:
Please use descriptive names for the controls, here eg `primarySortLabel`. This essentially applies to all controls on all tabs.